### PR TITLE
Abehjati/pyth-evm-contract-fire-event

### PR DESCRIPTION
**Pull request by ali-bahjati**
It also fixes a gas issue in evm relayer and deletes an unused open zeppelin file (for a test network)

### CODE REVIEW

File: ethereum/contracts/pyth/Pyth.sol [hunk 1]
```diff
@@ -39,6 +39,7 @@ contract Pyth is PythGetters, PythSetters, AbstractPyth {
 
             if(attestation.attestationTime > latestPrice.attestationTime) {
                 setLatestPriceInfo(attestation.priceId, newPriceInfo(attestation));
+                emit PriceUpdate(attestation.priceId, attestation.publishTime);
```
Comment (tompntn) [inline ethereum/contracts/pyth/Pyth.sol:42]: Do we need to declare the `PriceUpdate` event anywhere?

File: ethereum/contracts/pyth/Pyth.sol [hunk 2]
```diff
@@ -39,6 +39,7 @@ contract Pyth is PythGetters, PythSetters, AbstractPyth {
...
             }
```
Comment (tompntn) [inline ethereum/contracts/pyth/Pyth.sol:43]: Could we add a test for this?

**Review by tompntn — COMMENTED** [review]
Nice!

File: ethereum/contracts/pyth/Pyth.sol (hunk 1 above)
Comment (ali-bahjati) [reply ethereum/contracts/pyth/Pyth.sol:42]: It's in the solidity sdk.

Comment (drozdziak1) [inline ethereum/contracts/pyth/Pyth.sol:42]: Small nit: I noticed that you had to raise the gas limit, which seems like the event might cost a fair bit. Do we need the publish time inside the event? If we're already giving them the price ID, consumers can look the rest of the metadata up, which could make this method cheaper for relayers. My thinking is that smaller events may be cheaper (just a hunch).

Comment (ali-bahjati) [reply ethereum/contracts/pyth/Pyth.sol:42]: I am not sure it was working before or not. It is ok on 4 price feeds and not ok on 5 price feeds. I will try it out and will let you know.

The rationale behind having publish time is to make monitoring easier for us (specially I like to compare publish time vs block time)

Comment (drozdziak1) [reply ethereum/contracts/pyth/Pyth.sol:42]: That makes sense, we can always come back to optimize this.

Comment (ali-bahjati) [reply ethereum/contracts/pyth/Pyth.sol:42]: I just looked back and it doesn't work on 5 symbols batch in main branch. I also checked with an older change of switching from `updatePriceBatchFromVm` to `updatePriceFeeds` and it wasn't still working there. So I think this change to 200k is legit here.

**Review by tompntn — APPROVED** [review]

**Pull request merged**